yaml/degoogle.yml

```diff
@@ -16,9 +16,9 @@ web based products:
         [Issue #12](https://github.com/tycrek/degoogle/issues/12). Thanks @pydo, @ThijsRay, and @DatAres37.
         Also see [this comment in Issue #99](https://github.com/tycrek/degoogle/issues/99#issuecomment-616224650)
         from @danarel on Startpage.
-    - name: searx.me
-      url: https://searx.me/
-      repo: https://github.com/asciimoo/searx
+    - name: searx
+      url: https://searx.space/
```

Comment:
I would suggest changing the URL to [the Pages](https://searx.github.io/searx/) linked by the official repo

Reply:
Yeah. We could do that. But this makes it harder for the average user to just try the product, since we'd link the repository and the documentation.

My intention was to show give the average user a link to a running instance and since searx doesn't seem to host a main instance, I'd link to the oversight of all usable instances.

Your way might be the better one but the less user friendly...

I can change it if you like

Reply:
Ok cool it didn't keep my reply when I resolved, thanks GitHub!

With that info I do prefer your link 👍